packages/wow-combat-log-parser/package.json

```diff
@@ -2,7 +2,7 @@
   "name": "wow-combat-log-parser",
   "author": "WoW Arena Logs",
   "license": "MIT",
-  "version": "4.6.4",
+  "version": "5.0.0",
```

Comment:
wondering if we should align the major version of this package with major wow version or combat log version :)